src/prompt.ts

```diff
@@ -38,9 +38,12 @@ export const createPromptWindow = async () => {
   promptWindow = new BrowserWindow({
     frame: false,
     transparent: true,
-    backgroundColor: '#00000000',
+    backgroundColor: nativeTheme.shouldUseDarkColors
+      ? '#33000000'
+      : '#C0FFFFFF',
+    vibrancy: nativeTheme.shouldUseDarkColors ? 'dark' : 'medium-light',
     show: false,
-    hasShadow: false,
+    hasShadow: true,
```

Comment:
having a shadow makes it pop a bit more. [related issue](https://github.com/johnlindquist/kit/issues/128)